components/molecules/Table.js

```diff
@@ -0,0 +1,76 @@
+import { useContext } from "react";
+import { LanguageContext } from "../../context/languageProvider";
+import en from "../../locales/en";
+import fr from "../../locales/fr";
+
+function Table({ rows, columns }) {
+  return (
+    <table className="w-full cursor-text text-left">
+      <thead className="text-sm font-display">
+        <tr>
+          <th key={columns[0].heading} className="pl-6 pb-2">
+            {columns[0].heading}
+          </th>
+          <th key={columns[1].heading} className="text-center pb-2">
+            {columns[1].heading}
+          </th>
+        </tr>
+      </thead>
+      <tbody className="divide-y-2 divide-gray-light-100 border-2 border-gray-light-100">
+        {rows.map(({ text, subtext, link, href }, idx) => (
+          <tr key={`${text[idx]}`}>
```

Comment:
Can't "key" just be the index? (eg, `key={idx}`)

Reply:
Yeah, that's definitely better.